### PR TITLE
Create Android APK version of the agent

### PR DESCRIPTION
**Pull request by derekwbrown**
### What does this PR do?

Creates an android APK version of the agent

### Motivation

Customer request/promise

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 [[constraint]]
   name = "github.com/DataDog/gohai"
-  branch = "master"
+  branch = "db/android"
```
Comment (hush-hush) [inline Gopkg.toml]: To change before we merge.

File: tasks/agent.py [hunk 2]
```diff
@@ -44,14 +44,18 @@
 @task
 def build(ctx, rebuild=False, race=False, build_include=None, build_exclude=None,
           puppy=False, use_embedded_libs=False, development=True, precompile_only=False,
-          skip_assets=False):
+          skip_assets=False, android=False):
```
Comment (hush-hush) [inline tasks/agent.py]: Why do we need to update `agent.build` command if we have the `android.build` one ?

**Review by hush-hush — COMMENTED** [review]
Added a few questions. Knowing nothing to android this seems good !

File: tasks/agent.py (hunk 2 above)
Comment (derekwbrown) [reply tasks/agent.py]: good catch... I started with build, but decided to make its own command.  Now fixed.

**Review by hush-hush — APPROVED** [review]

**Pull request merged**